program-test/src/lib.rs

```diff
@@ -333,6 +334,55 @@ impl solana_sdk::program_stubs::SyscallStubs for SyscallStubs {
         stable_log::program_success(&logger, &program_id);
         Ok(())
     }
+
+    fn sol_get_sysvar(&self, id: &Pubkey, var_addr: *mut u8) -> u64 {
+        let invoke_context = get_invoke_context();
+
+        ic_msg!(invoke_context, "sol_get_sysvar");
+
+        let sysvars = match invoke_context.get_sysvar(id).ok_or_else(|| {
```

Comment:
Oh nice, this is much simpler than I expected, thanks for providing the example!